servers/rendering/renderer_rd/environment/gi.h

```diff
@@ -738,44 +736,52 @@ class GI : public RendererGI {
 	};
 
 	struct PushConstant {
-		uint32_t view_index;
 		uint32_t max_voxel_gi_instances;
 		uint32_t high_quality_vct;
 		uint32_t orthogonal;
+		uint32_t pad1;
 
 		float proj_info[4];
 
 		float z_near;
 		float z_far;
-		float pad1;
 		float pad2;
+		float pad3;
 	};
 
 	RID sdfgi_ubo;
+
```

Comment:
This looks ok, but at some point we seriously need to start using specialization constants for these things to reduce the amount of shader variations.

Reply:
Good point because I'm running into that for some of the other shaders too, I need to get some more experience using those

Reply:
Ended up using this for all settings, VRS, Projection matrix and half resolution. Also found and fixed a bug with specialisation constants and compute shaders.